### PR TITLE
make custom_ucr_expression module level setting

### PR DESCRIPTION
**Pull request by sravfeyn**
Some custom reports end up having lot of custom expressions, this lets us have neater groupings @snopoke 
FYI @czue

### CODE REVIEW

**Comment by snopoke** [comment]
LGTM

File: corehq/apps/userreports/expressions/__init__.py [hunk 1]
```diff
@@ -2,6 +2,15 @@
 from django.utils.module_loading import import_string
 from corehq.apps.userreports.expressions.factory import ExpressionFactory
 
+
+def get_custom_ucr_expressions():
+    custom_ucr_expressions = settings.CUSTOM_UCR_EXPRESSIONS
+
+    for expression_list in settings.CUSTOM_UCR_EXPRESSION_LISTS:
+        custom_ucr_expressions += import_string(expression_list)
```
Comment (czue) [inline corehq/apps/userreports/expressions/__init__.py]: is this not calling `import_string` on the whole list? also, even if that works won't the first half of the list be string tuples and the second half be imported modules? finally, if `CUSTOM_UCR_EXPRESSIONS` is a list and we add to it won't that get added over and over again each time?

```python
>>> x = [1, 2, 3]
>>> y = x
>>> y += [4, 5]
>>> x
[1, 2, 3, 4, 5]
>>> y += [4, 5]
>>> x
[1, 2, 3, 4, 5, 4, 5]
>>> 
```

Comment (sravfeyn) [inline corehq/apps/userreports/expressions/__init__.py]: I misnamed `expression_list` var. It is just a path that contains `settings.CUSTOM_UCR_EXPRESSIONS` like list

````python
In [11]: settings.CUSTOM_UCR_EXPRESSION_LISTS
Out[11]: 
['mvp.ucr.reports.expressions.CUSTOM_UCR_EXPRESSIONS',
 'custom.icds.ucr.expressions.CUSTOM_UCR_EXPRESSIONS']

In [12]: import_string(settings.CUSTOM_UCR_EXPRESSION_LISTS[0])
Out[12]: 
[('mvp_medical_cause', 'mvp.ucr.reports.expressions.medical_cause_expression'),
 ('mvp_no_treatment_reason',
  'mvp.ucr.reports.expressions.no_treatment_reason_expression'),
 ('mvp_treatment_provider_name',
  'mvp.ucr.reports.expressions.treatment_provider_name_expression'),
 ('mvp_treatment_place_name',
  'mvp.ucr.reports.expressions.treatment_place_name_expression'),
 ('mvp_death_place', 'mvp.ucr.reports.expressions.death_place_expression')]

Comment (czue) [inline corehq/apps/userreports/expressions/__init__.py]: ah, i see. the two levels of dynamic import indirection and variable names were confusing me. I think we should probably try and tidy it up a little bit (for one there is no need to do dynamic imports if the module where the expression functions are defined is defining the list), but makes more sense now.

I think the point about the adding to the list without copying still holds.

Comment (czue) [inline corehq/apps/userreports/expressions/__init__.py]: (cleanup bit not blocking)

**Comment by czue** [comment]
:+1: on :white_check_mark:

**Pull request merged**